share/examples/nrniv/nmodl/cagkftab.py

```diff
@@ -72,11 +72,13 @@ def memorder():
     rmat = mat._ref_x[0][0]
     ra = h._ref_a[0][0]
     for i in range(nrow * ncol):
+        # olupton 2022-11-22: this relies on indexing i elements past a scalar (mat[0][0]); this was allowed by T* but not by data_handle<T>
```

Comment:
@nrnhines note that this (new) test from https://github.com/neuronsimulator/nrn/pull/2037 doesn't work in this branch for the above reason. Should I just remove this `memorder()` part, or do you want to check that some other way?

Reply:
I re-enabled this test in 3422f3197, replacing some checks that indices `i` are zero with calls to `data_handle<T>::next_array_element(i)`. @nrnhines does this seem reasonable to you? Do I need to make the same change in more places?